go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/open-policy-agent/gatekeeper/v3
 
 go 1.21
 
+replace github.com/open-policy-agent/frameworks/constraint => /mount/d/go/src/github.com/open-policy-agent/frameworks/constraint
```

Comment:
Use latest frameworks once [frameworks#427](https://github.com/open-policy-agent/frameworks/pull/427) is merged